src/commands.rs

```diff
@@ -2579,7 +2579,7 @@ fn diff_summary_to_description(bytes: &[u8]) -> String {
         "Summary diffs and repo paths must always be valid UTF8.",
         // Double-check this assumption for diffs that include file content.
     );
-    "JJ: This commit contains the following changes:\n".to_owned()
+    "\nJJ: This commit contains the following changes:\n".to_owned()
```

Comment:
Perhaps, `.trim_end_matches('\n')` can be added to the caller's `commit.description()` to eliminate excessive newlines.

Another option is to wrap `commit.description()` with `cli_util::complete_newline()` (or similar function that doesn't require `&mut String`) instead of adding `\n` to the subsequent messages.